spec/index.bs

```diff
@@ -7229,27 +7229,40 @@ dictionary GPURenderPassColorAttachment {
 <div class=validusage dfn-for=GPURenderPassColorAttachment>
     <dfn abstract-op>GPURenderPassColorAttachment Valid Usage</dfn>
 
-    Given a {{GPURenderPassColorAttachment}} |this| the following validation rules
-    apply:
-
-    1. Let |renderTextureDesc| be |this|.{{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[texture]]}}.{{GPUTexture/[[descriptor]]}}.
-    1. Let |resolveTextureDesc| be |this|.{{GPURenderPassColorAttachment/resolveTarget}}.{{GPUTextureView/[[texture]]}}.{{GPUTexture/[[descriptor]]}}.
-    1. |this|.{{GPURenderPassColorAttachment/view}} must have a [=color renderable format=].
-    1. |renderTextureDesc|.{{GPUTextureDescriptor/usage}} must contain {{GPUTextureUsage/RENDER_ATTACHMENT}}.
-    1. |this|.{{GPURenderPassColorAttachment/view}} must be a view of a single [=subresource=].
-    1. If |this|.{{GPURenderPassColorAttachment/resolveTarget}} is not `null`:
-
-        1. |this|.{{GPURenderPassColorAttachment/view}} must be multisampled.
-        1. |this|.{{GPURenderPassColorAttachment/resolveTarget}} must not be multisampled.
-        1. |resolveTextureDesc|.{{GPUTextureDescriptor/usage}} must contain {{GPUTextureUsage/RENDER_ATTACHMENT}}.
-        1. |this|.{{GPURenderPassColorAttachment/resolveTarget}} must be a view of a single [=subresource=].
-
-        1. The dimensions of the [=subresource=]s seen by |this|.{{GPURenderPassColorAttachment/resolveTarget}}
+    Given a {{GPURenderPassColorAttachment}} |this|:
+
+    1. Let |renderViewDescriptor| be |this|.{{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.
+    1. Let |resolveViewDescriptor| be |this|.{{GPURenderPassColorAttachment/resolveTarget}}.{{GPUTextureView/[[texture]]}}.{{GPUTexture/[[descriptor]]}}.
+    1. Let |renderTextureDescriptor| be |this|.{{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[texture]]}}.{{GPUTexture/[[descriptor]]}}.
+    1. Let |resolveTextureDescriptor| be |this|.{{GPURenderPassColorAttachment/resolveTarget}}.{{GPUTextureView/[[texture]]}}.{{GPUTexture/[[descriptor]]}}.
+
+    The following validation rules apply:
+
+    - |renderViewDescriptor|.{{GPUTextureViewDescriptor/format}} must be a [=color renderable format=].
+    - |this|.{{GPURenderPassColorAttachment/view}} must be a [$renderable texture view$].
```

Comment:
looks like we have a bit of a problem - #2459
that is to say, we can still merge your stuff and follow-up